driver/web/docs/resources/admin/data-content-items.yaml

```diff
@@ -0,0 +1,84 @@
+post:
+  tags:
+    - Admin 
+  summary: Admin API that Creates a data content item
+  description: |
+    Creates a new data content item
+
+    **Auth:** Requires admin token with `all_admin_content` permission
+  security:
+    - bearerAuth: [] 
+  requestBody:
+     description: Creates a data content item
+     content:
+       application/json:
+         schema:
+           $ref: "../../schemas/apis/admin/data-content-item/DataContentItem.yaml"          
+  responses:
+    200:
+      description: Success
+    400:
+      description: Bad request
+    401:
+      description: Unauthorized
+    500:
+      description: Internal error 
+put:
+  tags:
+    - Admin
+  summary: Admin API that Updates data content item 
+  description: |
+    Updates data content item 
+
+    **Auth:** Requires admin token with `all_admin_content` permission
+  security:
+    - bearerAuth: [] 
+  requestBody:
+     description: Updates data content item
+     content:
+       application/json:
+          schema:
+            $ref: "../../schemas/apis/admin/data-content-item/DataContentItem.yaml"                         
+  responses:
+    200:
+      description: Success
```

Comment:
The docs should show that you are returning the updated content item in your handler.